web_client/src/store/index.ts

```diff
@@ -51,7 +51,7 @@ function prepareProxyManager(proxyManager) {
       view.getRepresentations().forEach((representation) => {
         representation.setInterpolationType(InterpolationType.NEAREST);
         representation.onModified(macro.debounce(() => {
-          view.render(true);
+          if (view.getRepresentations()) view.render(true);
```

Comment:
Let's always use curly braces with `if` statements (if possible we can add this to our eslint config, or whatever tool we are using for that now)